homeassistant/components/squeezebox/browse_media.py

```diff
@@ -81,11 +81,15 @@
     "New Music": MediaType.ALBUM,
 }
 
-BROWSE_LIMIT = 1000
-
 
 async def build_item_response(
-    entity: MediaPlayerEntity, player: Player, payload: dict[str, str | None]
+    entity: MediaPlayerEntity,
+    player: Player,
+    payload: dict[
+        str,
+        str | None,
+    ],
```

Comment:
```suggestion
    payload: dict[str, str | None],
```

Reply:
Ruff formatter made that change

Reply:
probably because of trailing comma after `None`, which causes ruff to add a carriage return
If you remove that trailing comma it should be happy to make it a one-liner

Reply:
Yep - you're right.